### ISSUE TITLE
The tab order is not logical in Tabs tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1924]
URL: http://clarity-ks.surge.sh/tabs

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Tabs tab.
3. Use tab key to access all interactive elements.

Actual Results:
The tab order is not logical after accessing “Management” menu in Tabs tab.
Tab order skips “Management Level1 Tab1” to go to “Cloud”.

Expected Results:
Tab order should go to “Management Level1 Tab1” after accessing “Management” and the secondary menus are expanded.

Violation of Section 508 Standards: 
1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind and disable with hand.

NOTE: We need to understand if this is an implementation issue or a Clarity issue.

**Comment by mathisscott** [comment]
@reddolan and @youdz 
Thoughts?

**Comment by mathisscott** [comment]
@reddolan ?

**Comment by reddolan** [comment]
For nested Tabs, we should navigate into the child tabs. Tabs are a type of navigation similar to Vertical Nav. If VertNav has nested links but are not shown, tabbing would navigate across the parent links. Only when a parent is expanded to show child links would we navigate through the child links. Tabs should behave like this as well. If nested tabs become visible, we need to update the tab order.

**Comment by Jinnie** [comment]
According to WC3 recommendations for tabs, the tab-list navigation should be reachable by Tab-key, but then focusing adjacent tabs should be done with the arrow keys:
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

Same behavior can be observed on the Mozilla Dev documentation of Tabs role:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

The only important decision left to the developers (us) is whether tab content will be displayed immediately (on focus), or with manual activation (space, Enter).
